src/arch/xtensa/configs/override/mt8186_chrome_waves.config

```diff
@@ -0,0 +1,3 @@
+CONFIG_CAVS_LPS=y
```

Comment:
Why need this for mt8186? It seems used to enable Intel cAVS Low Power Sequencer